apps/site/src/components/NavBar/NavBar.module.scss

```diff
@@ -37,20 +37,20 @@
 
 .homeActive {
 	@extend .text;
-	color: variables.$green;
+	color: utils.$green;
```

Comment:
Thought [repeating]: might make more sense to have theme-specific variables referenced as such and reserve utils for Bootstrap mixins, but I can also see convenience in having one central module. I suppose as long as we're careful to avoid name collisions, should still be okay.